src/views/htmlcontent/src/css/color-theme-dark.css

```diff
@@ -28,6 +28,10 @@ slick-grid.active .grid .slick-cell.selected {
   background-color: var(--color-cell-bg-grid-selected);
 }
 
+.slick-cell .grid-cell-value-container.missing-value {
+  color: var(--color-content) !important;
```

Comment:
- It's the default color for all other entries other than null, I believe, I can check again on chrome debugger.
- !important prioritizes the style given and takes precedence over any other tags which might be affecting that component. I had to put it in because otherwise it was retrieving style from SlickGrid.css, which is a generated .css file